SwiftWisdom/Core/NSData/NSData+Extensions.swift

```diff
@@ -20,8 +20,10 @@ public extension NSData {
         bufer.forEach {
             let hi  = Int(($0 & 0xf0) >> 4)
             let low = Int($0 & 0x0f)
-            output[ix++] = hexChars[ hi]
-            output[ix++] = hexChars[low]
+            ix += 1
```

Comment:
This isn't quite the right translation, it should be

```Swift
output[ix] = hexChars[hi]
ix += 1
output[ix] = hexChars[low]
ix += 1
```

For example:

```Swift
var a = 1
var arr = [1,2,3,4]
arr[a++]
```

Returns `2`, not `3`